tasks/update-granules-cmr-metadata-file-links/tests/test-index.js

```diff
@@ -169,3 +170,52 @@ test.serial('update-granules-cmr-metadata-file-links does not throw error if no
     () => updateGranulesCmrMetadataFileLinks(newPayload)
   );
 });
+
+test.serial('update-granules-cmr-metadata-file-links clears checksums for updated CMR file', async (t) => {
+  const newPayload = buildPayload(t);
+
+  await validateConfig(t, newPayload.config);
+  await validateInput(t, newPayload.input);
+
+  const filesToUpload = cloneDeep(t.context.filesToUpload);
+  await uploadFiles(filesToUpload, t.context.stagingBucket);
+
+  const message = await updateGranulesCmrMetadataFileLinks(newPayload);
+
+  message.granules.forEach((granule) => {
+    granule.files.forEach((file) => {
+      if (file.type === 'metadata') {
```

Comment:
I'm unclear how this distinction will happen in the code as written, it looks like file.checksum/file.checksumType is gonna be gone no matter what?

Reply:
@etcart  I believe that's only supposed to be the case for granule files of type 'metadata' (as a proxy for CMR files), which is what this is testing - that said, this should be more properly checked as not all metadata files are CMR files. 

9221ceada1e